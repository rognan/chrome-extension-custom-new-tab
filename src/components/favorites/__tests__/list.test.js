import React from 'react';
import List from '../list';
import { shallow } from 'enzyme';

describe('List', () => {

    it('should display heading', () => {
        const component = shallow(<List heading={'Foo'} items={['Bar']} /> );

        expect(component).toMatchSnapshot();
    });
});

