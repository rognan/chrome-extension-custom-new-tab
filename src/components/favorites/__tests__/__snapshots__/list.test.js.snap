// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should display heading 1`] = `ShallowWrapper {}`;
